watchdog_linux.go

```diff
@@ -22,14 +22,14 @@ func init() {
 	// returns the configured WatchdogSec in the service unit as time.Duration
 	interval, err := daemon.SdWatchdogEnabled(false)
 	if err != nil || interval == 0 {
-		log.Printf("Systemd watchdog not enabled")
+		log.Debug("Systemd watchdog not enabled")
```

Comment:
Most users won't need to see this an it get in the way when writing the json config to stdout with `--decrypt`.